### PR TITLE
Build out the Explore view

### PR DESCRIPTION
**Pull request by cnunciato**
This is backed by static data while we build the stats APIs.

Also adds support for storing HTML templates outside of component files.

Signed-off-by: Christian Nunciato <cnunciato@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
Fixes #2159.

**Comment by reset** [comment]
Closes #2128

File: support/Procfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+api: target/debug/bldr-api start --path /tmp/depot --config /vagrant/config_api.toml
 admin: target/debug/bldr-admin start
 router: target/debug/bldr-router start
 jobsrv: target/debug/bldr-job-srv start
 scheduler: target/debug/bldr-scheduler start
-sessionsrv: target/debug/bldr-session-srv start
+sessionsrv: target/debug/bldr-session-srv start --config /vagrant/config_sessionsrv.toml
 originsrv: target/debug/bldr-origin-srv start
-worker: target/debug/bldr-worker start
+worker: target/debug/bldr-worker start --config /vagarnt/config_worker.toml
```
Comment (reset) [inline support/Procfile]: Did you mean to include these changes in your PR?

Comment (reset) [inline support/Procfile]: Also vagrant is mispelled here ;)

Comment (cnunciato) [inline support/Procfile]: Ay, no -- thanks, totally unrelated and snuck in there.

**Comment by reset** [comment]
gif-keyboard-13303506388078103422

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/225389474).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/225389474).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**